railties/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "railties",
+  "private": true,
+  "dependencies": {
+    "@rails/actioncable": "^6.0.0-alpha",
+    "@rails/activestorage": "^6.0.0-alpha",
+    "@rails/ujs": "^6.0.0-alpha",
+    "@rails/webpacker": "^4.0.0-rc.7",
```

Comment:
What happens if we realease new versions of our packages? Or if there’s commits in them that haven’t yet been pushed to npm?

Reply:
Your concern is correct and we should not this approach. Since there was I wanted to confirm, I temporarily put this in.